inventory/idm-server/group_vars/all-aws.yml

```diff
@@ -0,0 +1,29 @@
+---
```

Comment:
Do you mean sample.osp.idm-server @tylerauerbeck ?

Reply:
@jimrigsbee Ahh yeah, osp, not ocp

Reply:
I do like the grouping suggestion, but I'd actually take it one step further. I.e.: we can just keep it all within one `idm-server` inventory, but have the `osp-provisioner` v.s. `aws-provisioner`, etc. (see other comments re: use of `localhost`). The inventory used for the actual IdM content is the same regardless, so the only difference would be for the provisioning part. This could then be split out into a group_vars for `aws_provisioner` and `osp_provisioner`. Such an example will showcase the power of ansible and inventories more than keeping a clone of the same inventory in multiple places.

Reply:
I have restructured the inventory into a single one as suggested.  I need a review before merge to make sure I've captured it the way you intend.